env/work/srw_python/srwlib.py

```diff
@@ -4167,6 +4207,8 @@ def srwl_wfr_emit_prop_multi_e(_e_beam, _mag, _mesh, _sr_meth, _sr_rel_prec, _n_
 
     #print('DEBUG MESSAGE: rank:', rank,': nPartPerProc=', nPartPerProc, 'nSentPerProc=', nSentPerProc, '_n_part_avg_proc=', _n_part_avg_proc)
 
+    log_path, total_num_of_particles = srwl_uti_save_stat_wfr_emit_prop_multi_e_init(rank, nProc, nPartPerProc, nSentPerProc, _n_part_avg_proc) #MR20012017
```

Comment:
The function for logs creation is called here - right after all required variables are defined:
- `rank`
- `nProc`
- `nPartPerProc`
- `nSentPerProc`
- `_n_part_avg_proc`